templates/shortcodes/references.md

```diff
@@ -1,14 +1,18 @@
 [appgallery]: https://appgallery.huawei.com/#/app/C104325611?local={{ lang }} "{{ trans(key='install-appgallery', lang=lang) }}"
+[appgallery_review]: appmarket://details?id=app.organicmaps
 [appstore]: https://apps.apple.com/app/organic-maps/id1567437057 "{{ trans(key='install-appstore', lang=lang) }}"
+[appstore_review]: https://itunes.apple.com/app/id1567437057?action=write-review
 [contributing]: https://github.com/organicmaps/organicmaps/blob/master/docs/CONTRIBUTING.md
 [email]: mailto:hello@organicmaps.app
+[email_support]: mailto:support@organicmaps.app
 [exodus]: https://reports.exodus-privacy.eu.org/en/reports/app.organicmaps/latest/
 [facebook]: https://facebook.com/OrganicMaps
 [fdroid]: https://f-droid.org/{{ lang }}/packages/app.organicmaps/ "{{ trans(key='install-fdroid', lang=lang) }}"
 [firebase]: https://appdistribution.firebase.dev/i/9ec3bca5e2b47373
 [flatpak]: https://flathub.org/apps/details/app.organicmaps.desktop
 [github]: https://github.com/organicmaps/organicmaps
 [googleplay]: https://play.google.com/store/apps/details?id=app.organicmaps&hl={{ lang }} "{{ trans(key='install-googleplay', lang=lang) }}"
+[googleplay_review]: market://details?id=app.organicmaps
```

Comment:
It took me a lot of energy to manually edit .po files in addition to markdown files in this case. Please modify and improve the workflow to avoid such situations in the future.